mono/metadata/custom-attrs.c

```diff
@@ -1107,6 +1148,155 @@ mono_reflection_create_custom_attr_data_args (MonoImage *image, MonoMethod *meth
 	*named_arg_info = NULL;
 }
 
+/*
+ * mono_reflection_create_custom_attr_data_args_noalloc:
+ *
+ * Same as mono_reflection_create_custom_attr_data_args_noalloc but allocate no managed objects, return values
+ * using C arrays. Only usable for cattrs with primitive/type arguments.
+ * TYPED_ARGS, NAMED_ARGS, and NAMED_ARG_INFO should be freed using g_free ().
+ */
+void
+mono_reflection_create_custom_attr_data_args_noalloc (MonoImage *image, MonoMethod *method, const guchar *data, guint32 len,
```

Comment:
This can either this call `mono_reflection_create_custom_attr_data_args` or can `mono_reflection_create_custom_attr_data_args` call this, or can they  be one function with more parameters?